students/eric_grandeo/lesson01/inventory_management/electric_appliances_class.py

```diff
@@ -0,0 +1,32 @@
+"""
+This is a module for Electric applicances.
+Classes:
+    electricAppliances: class for electric appliances
+"""
+from .inventory_class import Inventory
```

Comment:
No need for relative import here, since you are adding _inventory_management_ to your testing, so this one should simply be:

`from inventory_class import Inventory`